src/vs/workbench/contrib/output/browser/outputServices.ts

```diff
@@ -145,12 +150,21 @@ export class OutputService extends Disposable implements IOutputService, ITextMo
 		return null;
 	}
 
-	async showChannel(id: string, preserveFocus?: boolean): Promise<void> {
+	async showChannel(id: string, preserveFocus?: boolean, force?: boolean): Promise<void> {
```

Comment:
As described in the settings description, this should only block extensions from revealing the output channel so this should be in the `mainThreadOutputService`.

Reply:
Done